README.rst

```diff
@@ -188,11 +188,75 @@ Using CWL:
 
     $ cwltool --outdir=./outputs ./workflow/workflow.cwl ./workflow/input.yaml
 
+Create REANA file
+=================
+Putting all together, we can now describe our Higgs to 4 lepton physics analysis
+example, its runtime environment, the inputs, the code, the workflow and its
+outputs by means of the following REANA specification file:
+
+.. code-block:: yaml
+   
+    version: 0.2.0
+    metadata: 
+      authors:
+       - Diyaselis Delgado Lopez <diyaselis.delgado@upr.edu>
+      title: Higgs to 4 lepton physics analysis example
+      date: 22 September 2018
+      repository: https://github.com/reanahub/reana-demo-cms-h4l/
+    code: 
```

Comment:
... also the structure of `reana.yaml` was changed with the v0.3.0, e.g. there is now no distinction between "code" and "data" inputs in the YAML file. Can you please emulate the example from  https://github.com/reanahub/reana-demo-worldpopulation/  regarding YAML file and the running instructions?